AxonDeepSeg/download_model.py

```diff
@@ -14,9 +15,11 @@ def download_model(destination = None):
         tem_destination = destination / "default_TEM_model"
         model_seg_pns_bf_destination = destination / "model_seg_pns_bf"
 
-    url_TEM_model = "https://osf.io/2hcfv/?action=download&version=5"  # URL of TEM model hosted on OSF storage with the appropriate versioning on OSF
-    url_SEM_model = "https://osf.io/sv7u2/?action=download&version=5"  # URL of SEM model hosted on OSF storage with the appropriate versioning on OSF
-    url_model_seg_pns_bf = "https://osf.io/hfeu2/?action=download&version=2" # URL of Bright Field Optical Microscopy model hosted on storage with the appropriate versioning on OSF
+    url_TEM_model = "https://github.com/axondeepseg/default-TEM-model/archive/refs/tags/r20210615.zip" 
+    url_SEM_model = "https://github.com/axondeepseg/default-SEM-model/archive/refs/tags/r20210615.zip" 
+    url_model_seg_pns_bf = "https://github.com/axondeepseg/model-seg-pns-bf/archive/refs/tags/r20210615.zip"
```

Comment:
Is it retrieving the latest release? I believe hardcoding the **tag** name with the urlwould make us change those links in this script whenever we have a new release.

Reply:
No, this does not retrieve the latest release, but it is the behaviour we chose to ensure cross-compatibility and mostly reproducibility (e.g. if we want to update the models, it will require a PR on the ADS side)